src/latexgen.cpp

```diff
@@ -681,13 +681,26 @@ static QCString makeIndex()
   return result;
 }
 
+static QCString latex_batchmode()
+{
+  switch Config_getEnum(LATEX_BATCHMODE)
```

Comment:
This switch statement looks a bit weird without parenthesis; it probably works only because Config_getEnum is macro that already comes with the parenthesis.

Reply:
Indeed looks a bit strange and the macro (config_get...) already contains the brackets.

Updated.